codec-http/src/main/java/io/netty/handler/codec/http/websocketx/CloseWebSocketFrame.java

```diff
@@ -140,12 +140,22 @@ public String reasonText() {
 
     @Override
     public CloseWebSocketFrame copy() {
-        return new CloseWebSocketFrame(isFinalFragment(), rsv(), content().copy());
+        return (CloseWebSocketFrame) super.copy();
     }
 
     @Override
     public CloseWebSocketFrame duplicate() {
-        return new CloseWebSocketFrame(isFinalFragment(), rsv(), content().duplicate());
+        return (CloseWebSocketFrame) super.duplicate();
+    }
+
+    @Override
+    public CloseWebSocketFrame rduplicate() {
+        return (CloseWebSocketFrame) super.rduplicate();
+    }
+
+    @Override
+    protected CloseWebSocketFrame copy(ByteBuf content) {
```

Comment:
`copy` -> `newCloseWebSocketFrame`? copy nomenclature is overloaded with `ByteBuf`'s copy.